### PR TITLE
edited some wording in AO intro and link text

### PR DESCRIPTION
**Pull request by tcannon3**
Changed wording around experienced teacher and also shortened providers offering assessment only QTS to international teachers

### Trello card
https://trello.com/c/EPcL36d6/1859-revisit-the-content-at-the-top-of-the-assessment-only-page

### CODE REVIEW

File: app/views/content/assessment-only-providers.md [hunk 1]
```diff
@@ -502,14 +502,14 @@ keywords:
...
+If you have a degree and enough classroom experience, you may be able to get qualified teacher status (QTS) through a 12-week assessment only (AO) programme.
 
 The accredited teacher training providers listed on this page offer the AO programme.
 
 Contact them directly to ask them about entry criteria and to apply.
 
 ### Teachers from outside the UK
 
-If you trained as a teacher outside the UK, you can apply to certain training providers for an assessment only QTS without visiting or training in England. 
+If you have experience of teaching outside the UK, you can apply to certain training providers for an assessment only QTS without visiting or training in England.
```
Comment (gemmadallmandfe) [inline app/views/content/assessment-only-providers.md:513]: Is this definitely right @tcannon3 - that you only need experience, not training, as an international AO applicant?

(That might well be the case - just wanted to check!)

File: app/views/content/assessment-only-providers.md [hunk 2]
```diff
@@ -502,14 +502,14 @@ keywords:
   - experienced
 ---
 
-If you’re an experienced teacher with a degree, you may be able to get qualified teacher status (QTS) through a 12-week assessment only (AO) programme.
+If you have a degree and enough classroom experience, you may be able to get qualified teacher status (QTS) through a 12-week assessment only (AO) programme.
```
Comment (gemmadallmandfe) [inline app/views/content/assessment-only-providers.md:505]: Is this definitely right @tcannon3 - that we want to change from 'experienced' to 'enough classroom experience'?

(Might be the case - just wanted to check that it is accurate!)

**Comment by gemmadallmandfe** [comment]
I think the heading needs changing as well @tcannon3 - 'international' doesn't make sense as 'a region of the provider'?

Perhaps 'Select a region' would be better?

image

File: app/views/content/assessment-only-providers.md (hunk 2 above)
Comment (tcannon3) [reply app/views/content/assessment-only-providers.md:505]: Yes. According to UCAS and providers, the AO route is for those with substantial classroom experience. The current wording around experienced teachers makes it seem like this is a route for those who are already teachers in which case they would also already have QTS. 
Happy to revise the wording around this though if needed :)

File: app/views/content/assessment-only-providers.md (hunk 1 above)
Comment (tcannon3) [reply app/views/content/assessment-only-providers.md:513]: I read on TES that international AO applicants can be any member of school staff -unqualified teacher, TA or higher-level teaching assistant with significant teaching experience. Basically it's someone who can prove that they meet all of the Teaching Standards after having significant classroom experience in a British Overseas school

Comment (tcannon3) [reply app/views/content/assessment-only-providers.md:513]: Having said that, I've just double checked their entry requirements and it seems that they want to international applicant to be employed in a teaching role within a school. But that doesn't necessarily mean trained teacher as the candidate could be an unqualified teacher

Comment (tcannon3) [reply app/views/content/assessment-only-providers.md]: @gemmadallmandfe Phoebe has given the OK for these changes

File: app/views/content/assessment-only-providers.md (hunk 2 above)
Comment (tcannon3) [reply app/views/content/assessment-only-providers.md:505]: @gemmadallmandfe Phoebe has OK'd these changes also

**Review by Tomcowan1 — APPROVED** [review]

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**